cypress_tests/package-lock.json

```diff
@@ -1,9507 +1,8 @@
 {
     "name": "cypress",
     "version": "1.0.0",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
Think this will cause problems updating the lockfile version? Not sure why this has been updated?

Reply:
I didn't realise I'd changed this, I'll change it back 👍